### ISSUE TITLE
Controller support missing

### ISSUE THREAD

**Issue by siteod** [svkaiser/Doom64EX#88]
In the 2.5 build on the wordpress site, controller support is included. But in the git build, it is missing from menus and the game doesn't read any input if you try using one anyways

**Comment by dotfloat** [comment]
Yes, the controller support is missing in this version. It's sort of working in the `rom` branch (about as well as in 2.5) but I think it can be done better.

I'll close this once I merge the `rom` branch and release some builds.
